### PR TITLE
make update manager work with overlays

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

#715

## This introduces a breaking change

- [ ] Yes
- [x] No

I need to test this on jupyternb-requirements or some repo with overlays to see if the feature is working. It is still working on flat repos though.

### CODE REVIEW

File: kebechet/managers/update/messages.py [hunk 1]
```diff
@@ -156,8 +156,7 @@
 
 Make sure your `Pipfile` or `requirements.in` or `requirements-dev.in` is placed in the root of your Git repository.
 
-/kind feature
-/priority important-soon
+/kind feature /priority important-soon
```
Comment (pacospace) [inline kebechet/managers/update/messages.py:159]: this needs to be on two different lines or they will all be considered /kind I think

File: kebechet/managers/update/update.py [hunk 2]
```diff
@@ -813,62 +854,109 @@ def run(self, labels: list) -> typing.Optional[dict]:
...
+            results: dict = {}
+
+            for e in runtime_environments:
+                close_no_management_issue = partial(
+                    self.close_issue_and_comment,
+                    _ISSUE_NO_DEPENDENCY_NAME.format(environment_name=e),
+                    comment=ISSUE_CLOSE_COMMENT.format(sha=self.sha),
+                )
+                with cwd(thoth_config.get_overlays_directory(e)):
+                    print(os.getcwd())
```
Comment (pacospace) [inline kebechet/managers/update/update.py:892]: print?

**Review by pacospace — COMMENTED** [review]
lgtm! 💯 👏🏼

**Review by harshad16 — APPROVED** [review]
lgtm 
thanks :+1:

**Comment by pacospace** [comment]
> ## Related Issues and Dependencies
> #715
> 
> ## This introduces a breaking change
> * [ ]  Yes
> * [x]  No
> 
> I need to test this on jupyternb-requirements or some repo with overlays to see if the feature is working. It is still working on flat repos though.

best repo could be the `ps-` repos that have overlays @KPostOffice

**Review by pacospace — APPROVED** [review]

**Pull request merged**